roles/callhome/node/tasks/install.yml

```diff
@@ -55,9 +55,12 @@
 - include_tasks: install_{{ scale_installmethod }}.yml
 
 - debug:
-        msg: "{{ scale_install_all_packages }}"
+    msg: "{{ scale_install_all_packages }}"
 
-- meta: flush_handlers
+- name: yum-clean-metadata
+  command: yum clean metadata
+  args:
+   warn: false
```

Comment:
What's the reason for this change? It seems to be unrelated to the other changes and unrelated to the daemon network config... Also, disabling warnings should only be used in very rare cases.

Reply:
We got warnings at the screen, that the meta-handler isn't working because of a include statement .

[WARNING]: flush_handlers task does not support when conditional

As we moved it to here the warning was gone.
If you have a better solution let me know.

Reply:
That's a good catch - thanks for spotting it!

I'd suggest to open a separate issue for it so we can discuss in more detail. This seems to be a general problem with the structure of the callhome role (conditional import of the file `install.yml`).

I don't think that replacing the handler with a plain command is appropriate to address this structural issue.

Reply:
I've opened #251 to document the issue.